src/components/pages/CaseDetails/RenderCaseDetails.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import { Layout, Card } from 'antd';
+import NavHeader from '../../common/NavHeader';
+import BreadCrumbs from '../../common/BreadCrumbs';
+import { CaseDetailsCollapse } from '../../common/CaseDetailsCollapse';
+const { Header, Content, Footer } = Layout;
+
+const RenderCaseDetails = () => {
+  const location = useLocation();
+
+  return (
+    <div className="RenderLayoutTemplate">
+      <Layout className="Layout">
```

Comment:
You implemented the Ant Design Layout really well. It matches our template has been expanded through your needs for the component.